src/js/providers/html5.js

```diff
@@ -318,6 +321,32 @@ function VideoProvider(_playerId, _playerConfig, mediaElement) {
         }
     }
 
+    function checkStartDateTime() {
+        if (_videotag.getStartDate) {
+            const startDate = _videotag.getStartDate();
+            const startDateTime = startDate.getTime();
+            if (startDateTime !== _this.startDateTime && !isNaN(startDateTime)) {
+                _this.startDateTime = startDateTime;
+                const programDateTime = startDate.toISOString();
+                const { start, end } = _this.getSeekRange();
+                const metadataType = 'program-date-time';
+                const metadata = {
+                    metadataType,
+                    programDateTime,
+                    start,
+                    end
+                };
+                const cue = _this.createCue(start, end, JSON.stringify(metadata));
+                _this.addVTTCue({
```

Comment:
can i get a quick ELI5 on what cues do in this instance? i went and looked at the underlying function and my assumption is it's heavily reliant on the `metadata` flag but i'm not quite getting it. we don't show these in the ui right? is it a way of cueing up the information for the events we're sending in safari?

Reply:
>  is it a way of cueing up the information for the events we're sending in safari?

Exactly. The video element has audio, video and text tracks. These text tracks can have a type of subtitles, chapters or metadata. These text tracks are made up of timed cues. When playback enters or exits a cue's range an event is fired notifying us that `activeCues` have changes. We use these events on the metadata track to fire "meta" events.